int32 label
string object_name
float32 probability
